mifosng-android/src/main/java/com/mifos/mifosxdroid/online/runreports/reportcategory/ReportCategoryPresenter.java

```diff
@@ -97,8 +97,7 @@ public void onNext(List<ClientReportTypeItem> clientReportTypeItems) {
         for (ClientReportTypeItem item : list) {
             map.put(item.getReportId(), item);
         }
-        ArrayList<ClientReportTypeItem> uniques = new ArrayList<>();
-        uniques.addAll(map.values());
+        ArrayList<ClientReportTypeItem> uniques = new ArrayList<>(map.values());
```

Comment:
Why and what makes it different from the existing code? is there any benefit to it?

Reply:
Just a refactor.

Reply:
Nice, but I think there are hundreds of line code like this in the entire project if we raise PR for each one of them separately and even if there is no benefit of it or without any specific reason, don't you think the number of PRs increases too much and it will be difficult and time-consuming to maintainers or mentors to merge them or because of this they might miss or delay to merge the important PRs?

Reply:
Ok Jawid, I will keep that in mind in future. Thankyou :)